src/OpenLoco/src/Map/StationElement.h

```diff
@@ -37,6 +37,12 @@ namespace OpenLoco::World
             _5 &= ~0x1F;
             _5 |= objectId & 0x1F;
         }
+        uint8_t unk5SHR5() const { return _5 >> 5; }
```

Comment:
this is just duplicating stationType and setStationType

Reply:
Argh, thanks. I must've missed them because they're in in StationElement.cpp.